deployment/deploy.sh

```diff
@@ -57,7 +64,7 @@ function create_bq_ops_dataset() {
 
 function create_bucket() {
     start_message "Creating bucket ${BUCKET_NAME}..."
-    gsutil mb -p $GOOGLE_CLOUD_PROJECT -l $LOCATION gs://$BUCKET_NAME
+    gsutil mb -p $GOOGLE_CLOUD_PROJECT -l $REGION gs://$BUCKET_NAME
```

Comment:
Since the create_ops_dataset references $LOCATION and buckets like datasets may be in either an entire location or region, perhaps we should use $LOCATION instead of $REGION here. That way, one may set LOCATION to US or us-central1 and that'd be perfectly fine. What do you think?